assets/components/SearchBar.js

```diff
@@ -1,9 +1,18 @@
 export const SearchBarSuggestionCard = ({ properties }) => {
   return `
     <div
-      class="sb-sugguestion--item" 
-      data-value='${ JSON.stringify({ lat: properties?.lat , lng: properties?.lon }) }'>
-      ${ properties?.address_line1 }, ${ properties?.address_line2 }
+      class="sb-suggestion-item"
+        data-addr1='${ properties?.address_line1 }'
+        data-latlng='${ JSON.stringify({ lat: properties?.lat , lng: properties?.lon }) }'>
```

Comment:
is this oke ` properties?.lon` or should it be ` properties?.lng`

Reply:
That `properties` object comes from API for `geocoding`. The key for longitude is `lon`, To keep it even in our application I started using `lng`.